gflownet/envs/base.py

```diff
@@ -891,18 +934,18 @@ def equal(state_x, state_y):
             return state_x == state_y
 
     @staticmethod
-    def isclose(state_x, state_y):
+    def isclose(state_x, state_y, atol=1e-8):
```

Comment:
Is there a reason you specify `atol` right now? I tried searching for some usage, but couldn't find any.

Reply:
Because at some point during development (bug investigation) I used it. Then I found an alternative way but thought this could be used at some point, so I decided to leave it. Would you remove it? For example, some envs might need more or less tolerance and could benefit from this.

Reply:
> Would you remove it?

Oh, no, I was just curious how why did you change that (and worried that maybe I missed something when I couldn't find any usage).